vlib/v/help/build/build-c.txt

```diff
@@ -180,36 +180,29 @@ see also `v help build`.
       more performant without autofree.
 
    -gc <mode>
-      Use and link an optional garbage collector. Only the Boehm–Demers–Weiser
-      garbage collector is supported currently with the following sub-options:
+      V uses a garbage collector by default. Only the Boehm–Demers–Weiser garbage collector
+      is supported currently with the following sub-options:
 
-      `-gc boehm` ........... selects the default mode for the architecture
+      `-gc boehm` ........... use default garbage collector (same as not specifying option)
       `-gc boehm_full` ...... full garbage collection mode
       `-gc boehm_incr` ...... incremental/generational garbage collection mode
       `-gc boehm_full_opt` .. optimized full garbage collection mode
       `-gc boehm_incr_opt` .. optimized incremental/generational garbage collection mode
       `-gc boehm_leak` ...... leak detection mode
+      `-gc none` ............ no garbage collector
 
-      You need to install a `libgc-dev` package first, or install it manually from:
-
-          https://github.com/ivmai/bdwgc
-
-      On Mac OS, it can be installed using homebrew (https://homebrew.sh/) with
-      `brew install libgc`.
-
-      Note, `-gc boehm` is complementary to -autofree. The Boehm garbage
-      collector is conservative, and it may make your program significantly
-      slower if it does many small allocations in a loop. This option
-      is intended *mainly* for reducing the memory usage of programs, that
-      process large amounts of text in *batch mode* on low/limited memory
-      environments like small VPSes, and for which a few ms of garbage
-      collection pauses from time to time *do not matter much*.
+      Note, the garbage collector is complementary to -autofree. The Boehm garbage
+      collector is conservative, and it may affect program speed if it does many small
+      allocations in a loop.
```

Comment:
Something like these, may be added at the end too:
```
      Note: some short lived applications, like compilers and other CLI tools can be
      more performant, without autofree, and without a garbage collector, instead
      relying on the OS to free the allocated memory automatically, after their 
      process ends.
      
      On most supported platforms, the source of the used garbage collector is
      located in thirdparty/libgc/, produced by post processing
      https://github.com/ivmai/bdwgc . If you have installed libgc by other means,
      (through your distro's package manager or by compiling it yourself and 
      installing it), you can also tell V to use it instead of its own copy,
      by passing `-d dynamic_boehm` too, while compiling your program.
      On Debian you can use `sudo apt-get install libgc-dev`. 
      On macOS, you can install it, using homebrew (https://homebrew.sh/) 
      with `brew install libgc`.
```